app/presenters/activity_presenter.rb

```diff
@@ -101,8 +101,12 @@ def linked_email
   end
 
   def render_partial
+    # Models from plugins are namespaced, e.g. "Dradis::Plugins::ModelName"
+    # and activity.trackable_type.underscore will return "dradis/plugins/model_name".
+    # So we demodulize it first to return "model_name" before passing in as locals.
+    trackable_name = activity.trackable_type.demodulize.underscore.to_sym
```

Comment:
Why is this desirable? Shouldn't the template for an activity live where the activity / trackable was created?

If the plugin plugin isn't installed, then the activity wouldn't be created / shouldn't be rendered?

Reply:
The template does live in the plugin. See https://github.com/secroots/dradispro-remediationtracker/pull/53/files#diff-776ca258a427b37c8180e81736583ff0710cafcc5bbeb14acd1885fcdd5dd318.

It's because using the previous code, it returns the symbol `:"dradis/pro/remediationtracker/ticket"`, and can't be accessed as `ticket` in the template.